<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CourtCaseWindow</class>
 <widget class="QWidget" name="CourtCaseWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>721</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="background" native="true">
     <property name="styleSheet">
      <string notr="true">#background {
	background-color: rgb(0, 0, 0, 40%);
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QWidget" name="mainWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>549</width>
          <height>350</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>549</width>
          <height>350</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#mainWidget {
	image: url(:/courtcasewindow/images/courtcasewindow/first-window.png)
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="closeLayout">
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QToolButton" name="close">
             <property name="minimumSize">
              <size>
               <width>85</width>
               <height>67</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">#close {
	image: url(:/courtcasewindow/images/courtcasewindow/close.png);
	border: none
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>38</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="twinceBauttons">
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>1</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="firstPageButton">
             <property name="minimumSize">
              <size>
               <width>53</width>
               <height>53</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">image:  url(:/courtcasewindow/images/courtcasewindow/first-page-button.png);
border: none
</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>120</width>
               <height>1</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="secondPageButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>46</width>
               <height>46</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">image: url(:/courtcasewindow/images/courtcasewindow/second-page-button.png);
border: none
</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>1</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QStackedWidget" name="stackedWidget">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="page">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>20</number>
             </property>
             <property name="topMargin">
              <number>7</number>
             </property>
             <property name="rightMargin">
              <number>20</number>
             </property>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <property name="spacing">
                <number>5</number>
               </property>
               <property name="topMargin">
                <number>1</number>
               </property>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <spacer name="horizontalSpacer_8">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeType">
                    <enum>QSizePolicy::Fixed</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>10</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">font-size: 14px;
color: #8da3c3;</string>
                   </property>
                   <property name="text">
                    <string>Выберите шаблон</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <property name="spacing">
                  <number>15</number>
                 </property>
                 <item>
                  <widget class="QComboBox" name="comboBox">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">QComboBox {
	background-color: #afc0da;
    border: none;
    border-radius: 20px;
	font-size: 14px;
}
QComboBox:on {
    padding-left: 6px;
}
QComboBox:!on {
    padding-left: 6px;
}
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top left;
    width: 18px;
	margin-left: 10px;
	image: url(:/courtcasewindow/images/courtcasewindow/combobox-button.png);
}
QComboBox:item {
       padding-left: 20px;  /* move text right to make room for tick mark */
}
QComboBox QAbstractItemView {
       border: none;
       color: black;
       selection-background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #111, stop: 1 #333);
}
QComboBox QListView {
	width: 20px;
}</string>
                   </property>
                   <property name="currentText">
                    <string/>
                   </property>
                   <item>
                    <property name="text">
                     <string/>
                    </property>
                   </item>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton_2">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>25</width>
                     <height>25</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">image: url(:/courtcasewindow/images/courtcasewindow/редактирование.png);
border: none;</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="page_2">
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QStackedWidget" name="stackedWidget_2">
               <widget class="QWidget" name="page_3"/>
               <widget class="QWidget" name="page_4"/>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="buttonLayout"/>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="okeyButtun">
             <property name="minimumSize">
              <size>
               <width>90</width>
               <height>65</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">#okeyButtun {
	image:  url(:/courtcasewindow/images/courtcasewindow/ok.png);
	border: none
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>322</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
